src/utils.js

```diff
@@ -17,6 +17,10 @@ import {Vector3} from 'three';
 
 import {HAS_FULLSCREEN_API, HAS_WEBXR_DEVICE_API, HAS_WEBXR_HIT_TEST_API, IS_AR_CANDIDATE} from './constants.js';
 
+export const domPointToArray = point => {
+  return new Float32Array([point.x, point.y, point.z]);
```

Comment:
It seems marginally wasteful to create this object (sometimes more than once) on every frame (which will happen once reticle is updated to use this).

Reply:
Yeah -- actually this can be handled in `castRay` which can cache those arrays too